### ISSUE TITLE
A11y_OSPO Production_Code of Conduct - FAQ_AI4W: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (span)

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/opensource.microsoft.com#219]
### Github tags:
#A11yMAS; #A11ySev2; #A11yTCS; #Benchmark; #P2_OSPOProduction_Web_Sep2021; #P2_WCP; #WCP; #OSPO Production; #DesktopWeb; #ChromiumEgde; #Win11; #P2_OSPOProduction; #WCAG1.4.3; #color-contrast;

### Environment and OS Details:
#URL: https://opensource.microsoft.com/codeofconduct/faq/
App name: OSPO Production
Windows 11 Version 21H2(OS Build 22000.795)
Chromium Edge Version 103.0.1264.62 (Official build) (64-bit)

### Note:
This issue is also observed in https://opensource.microsoft.com/program/

### Issue: 
Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

### Target application: 
Code of Conduct - FAQ | Microsoft Open Source - https://opensource.microsoft.com/codeofconduct/faq/

### Element path: 
li:nth-child(2) > span

### Snippet: 
span

### How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 1.46 (foreground color: #323130, background color: #11100f, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Microsoft Edge version 103.0.1264.71

====

This accessibility issue was found using Accessibility Insights for Web 2.32.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### Attachments:
![Code of Conduct - FAQ_AI4W](https://user-images.githubusercontent.com/83570638/181730125-1b2fb5e1-178e-4014-8167-7d10b1351691.png)
![Microsoft’s Open Source Program_AI4W_color-contrast](https://user-images.githubusercontent.com/83570638/181730168-024515f2-d543-4d60-94ae-72fd65437ead.png)

**Comment by mpcen** [comment]
Non-issue once list items are expanded.

**Comment by mpcen** [comment]
fixed in https://github.com/microsoft/opensource.microsoft.com/pull/236

**Comment by akashelijah05** [comment]
Verified the issue in https://opensource.microsoft.com/codeofconduct/faq/ and the issue is not fixed. Please reopen the bug, please find the screen shot for reference
![219_Not fixed](https://user-images.githubusercontent.com/116360471/198990722-845db605-be4b-4b57-a796-24686bb67364.jpg)

Verified the similar issue in https://opensource.microsoft.com/program/ and the issue is fixed in here only.

![219_Fixed_SImilar issue](https://user-images.githubusercontent.com/116360471/198991272-ef4e9b08-8e95-49fe-ab2d-448c843f1380.jpg)

**Comment by mpcen** [comment]
> The issue is partially fixed, please reopen the bug. Verified the issue in https://opensource.microsoft.com/codeofconduct/faq/ and the issue is not fixed, please find the screen shot for reference ![219_Not fixed](https://user-images.githubusercontent.com/116360471/198990722-845db605-be4b-4b57-a796-24686bb67364.jpg)
> 
> Verified the similar issue in https://opensource.microsoft.com/program/ and the issue is fixed in here only.
> 
> ![219_Fixed_SImilar issue](https://user-images.githubusercontent.com/116360471/198991272-ef4e9b08-8e95-49fe-ab2d-448c843f1380.jpg)

Have you tried collapsing the items?

**Comment by poojapatil36** [comment]
Verified the issue, issue is fixed.
Please refer below attachment.
Added tag: #Closed;
![219_A11y_OSPO Production_Code of Conduct - FAQ_AI4W (2)](https://user-images.githubusercontent.com/97668775/218019814-6d5162b7-b50f-4066-a06b-9aa14b4b3ac4.png)
![219_A11y_OSPO Production_Code of Conduct - FAQ_AI4W](https://user-images.githubusercontent.com/97668775/218019841-355ec8a6-d7f0-4259-b01a-61eb7cb367c0.png)
